spinalcordtoolbox/utils/sys.py

```diff
@@ -80,6 +80,14 @@ def _format(record):
         # Display SCT version
         logger.info('\n--\nSpinal Cord Toolbox ({})\n'.format(__version__))
 
+        # Display command (Only if called from CLI: check for .py in first arg)
+        # Use next(iter()) to not fail on empty list (vs. sys.argv[0])
+        if '.py' in next(iter(sys.argv), None):
+            script = os.path.basename(sys.argv[0]).strip(".py")
+            arguments = ' '.join(sys.argv[1:])
+            logger.info(f"Command run: {script} {arguments}\n"
```

Comment:
I would personally remove the "Command run:". People will quickly get it and it adds superfluous text.
```suggestion
            logger.info(f"{script} {arguments}\n"
```

Reply:
Good point. Also, it makes it harder to copy and paste the command by double-clicking. :stuck_out_tongue: